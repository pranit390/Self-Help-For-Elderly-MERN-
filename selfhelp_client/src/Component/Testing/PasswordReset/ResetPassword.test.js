import React from 'react';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
Enzyme.configure({ adapter: new Adapter() });
import {shallow} from 'enzyme'

import ResetPasswordScreen from '../../PasswordReset/ResetPassword.js';

describe("Reset password ", () => {
    it("should render my component", () => {
      const wrapper = shallow(<ResetPasswordScreen/>);
      expect(wrapper.getElements()).toMatchSnapshot();
    });
  });