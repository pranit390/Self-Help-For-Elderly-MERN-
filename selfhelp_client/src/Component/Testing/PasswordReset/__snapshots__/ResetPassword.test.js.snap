// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reset password  should render my component 1`] = `
Array [
  <div
    className="resetpassword-screen"
  >
    <form
      className="resetpassword-screen__form"
      onSubmit={[Function]}
    >
      <h3
        className="resetpassword-screen__title"
      >
        Forgot Password
      </h3>
      
      
      <div
        className="form-group"
      >
        <label
          htmlFor="password"
        >
          New Password:
        </label>
        <input
          autoComplete="true"
          id="password"
          onChange={[Function]}
          placeholder="Enter new password"
          required={true}
          type="password"
          value=""
        />
      </div>
      <div
        className="form-group"
      >
        <label
          htmlFor="confirmpassword"
        >
          Confirm New Password:
        </label>
        <input
          autoComplete="true"
          id="confirmpassword"
          onChange={[Function]}
          placeholder="Confirm new password"
          required={true}
          type="password"
          value=""
        />
      </div>
      <button
        className="btn btn-primary"
        type="submit"
      >
        Reset Password
      </button>
    </form>
  </div>,
]
`;
